README.md

```diff
@@ -12,208 +12,14 @@ Planned functionality for first release include:
-
-```js
-const message = {
-  to: '0x88a5C2c290d9919e46F883EB62F7b8Dd9d0CC45b',
-  data: '0x',
-  value: '1000000000'
-};
-const transactionId = await sdk.execute(identityAddress, message, privateKey);
-```
-
-The function takes three arguments:
-
-- `identityAddress` - address of that identity that requests execution
-- `message` - a message, in the same format as ethereum transaction, to be executed by the relayer
-- `privateKey` - a private key to be used to sign the transaction, the corresponding public key address needs to be connected to identity
-
-The function will return one result:
-
-- `transactionId` (also called `nonce`) - an id of execution, might be used to confirm excution
-
-To confirm transaction
-
-```js
-await sdk.confirm(identityAddress, transactionId, privateKey);
-```
-
-### Events
-
-To subscribe to an event:
-
-```js
-const callback(event) = {};
-await sdk.subscribe(identityAddress, 'eventType', callback)
-```
-
-Possible event names are: `KeyAdded`, `KeyRemoved`, `ConnectionRequested`.
-
-To unsubscribe to an event:
-
-```js
-await sdk.unsubscribe(identityAddress, 'eventType', callback);
-```
-
-### Key management
-
-To add a key:
-
-```js
-const transactionId = await sdk.addKey(identityAddress, publicKey, privateKey);
-```
-
-To remove a key:
-
-```js
-const transactionId = await sdk.removeKey(
-  identityAddress,
-  publicKey,
-  privateKey
-);
-```
-
-Generate and request a new key to be added to an existing identity:
-
-```js
-const [privateKey, identityAddress] = await sdk.connect('alex.ethereum.eth');
-```
-
-This function will generate a new private key and send a request to relayer to add a key to identity. The request needs to be confirmed from public key connected to identity at hand.
-
-The function takes one parameter:
-
-- `ensName` - ENS name of an existing identity
-
-and returns:
-
-- `privateKey` - newly generated private key to be stored on a local device in the most secure way possible
-- `identityAddress` - address of identity pointed to by given ENS name
-
-The function will throw:
-
-- `InvalidIdentity` - if Identity does not exist (i.e. ENS Name does not resolve or resolves to a non-identity address)
-
-Confirmation connection (when request event occurs):
-
-```js
-await identity.subscribe('ConnectionRequested', event => {
-  identity.addKey(event.key, firstPrivateKey);
-});
-```
-
-## Relayer
-
-To start relayer programmatically:
-
-```js
-import Relayer from '../../lib/relayer/relayer';
-const privateKey = '0x.....';
-const relayer = new Relayer('https://etherscan.io/{yourapikey}', privateKey);
-relayer.start();
-```
-
-To stop relayer:
-
-```js
-relayer.stop();
-```
-
-To start relayer from command line:
-
-```sh
-npx start-relayer [config-file]
-```
-
-Where config file is a JSON file in the following format:
-
-```js
-{
-  privateKey: '0x123...123',
-  port: 3311
-}
-```
-
-Default port is `3311`, `privateKey` is the only required field.
-
-## Running example on dev environment (quick option)
```

Comment:
I would keep this section on the main readme, as it tells a new user in 3 command lines how to run the demo